dias/analyzers/thermaldata_analyzer.py

```diff
@@ -200,7 +200,9 @@ def _find_longest_stretch(self, phase, freq, step=None, tol=0.2):
                 abs(this_stepdiff) < step * (1.0 - tol),
             )
             # Ensure the step is not only the right length but the right direction
-            bad_step = bad_step or (np.sign(this_stepdiff) != np.sign(curr_stt_stepdiff))
+            bad_step = bad_step or (
+                np.sign(this_stepdiff) != np.sign(curr_stt_stepdiff)
+            )
 
```

Comment:
Haha, this is why this edit was so important! I was staring at this for a while and did not see the brackets surrounding the latter half of the equation.